Marlin/src/HAL/shared/Delay.h

```diff
@@ -58,6 +58,10 @@
 
   #else
 
+    #ifndef DELAY_CYCLES_ITERATION_COST
+      #define DELAY_CYCLES_ITERATION_COST 6
```

Comment:
Cortex M0 as no refill cycles so previous code was correct. Don't know if `DELAY_CYCLES_ITERATION_COST` is defined for such boards otherwise this may realize longer delays

Reply:
reverted back to 4.